pandas/tests/extension/relobject/test_relobject.py

```diff
@@ -0,0 +1,268 @@
+import operator
+
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+
+from pandas.tests.extension import base
+
+from .array import RelObjectDtype, RelObjectArray, RelObj, make_data
+
+
+@pytest.fixture
+def dtype():
+    return RelObjectDtype()
+
+
+@pytest.fixture
+def data():
+    return RelObjectArray(make_data())
+
+
+@pytest.fixture
+def data_missing():
+    # Since _can_hold_na is False, we don't have missing values
+    return RelObjectArray([RelObj(10), RelObj(5)])
+
+
+@pytest.fixture
+def data_for_sorting():
+    return RelObjectArray([RelObj(4), RelObj(5), RelObj(3)])
+
+
+@pytest.fixture
+def data_missing_for_sorting():
```

Comment:
Ah, I clearly didn't have arrays that don't support NA in mind when I wrote this :)

Could you add this note to the `data_missing_for_sorting` docstring (in `tests/extensions/conftest.py`).?